spec/docs/graphmap.md

```diff
@@ -0,0 +1,52 @@
+# Assigning Triples to Named Graphs
+
+Each triple generated from an [=RML mapping=]() is placed into one or more graphs of the [=output dataset=](). Possible target graphs are the unnamed [=default graph=](), the [=IRI=]()-named [=named graphs=](), and [=blank node=]() named [=named graphs=]()
+
+Any [=subject map=]() or [=predicate-object=]() map MAY have one or more associated graph maps. They are specified in one of two ways:
+
+1. using the rml:graphMap property, whose value MUST be a [=graph map=](),
+2. using the [=constant shortcut property=]() rml:graph.
+
+Graph maps are themselves [=term maps=](). When [=RDF triples are generated=](), the set of target graphs is determined by taking into account any graph maps associated with the subject map or predicate-object map.
+
+If a [=graph map=]() generates the special IRI rml:defaultGraph, then the target graph is the [=default graph=]() of the [=output dataset=]().
+
+In the following [=subject map=]() example, all generated RDF triples will be stored in the named graph ex:DepartmentGraph.
+
+```turtle
+[]  rml:subjectMap [
+      rml:template "http://data.example.com/department/{DEPTNO}";
+      rml:graphMap [ rml:constant ex:DepartmentGraph ];
+].
+```
```

Comment:
Examples should be formatted like other examples in spec. Using `<aside ...>`

Reply:
done